sdk/textanalytics/ai-text-analytics/swagger/README.md

```diff
@@ -550,3 +472,120 @@ directive:
     where: $.definitions.HealthcareAssertion
     transform: $.description = "An object that describes metadata about the healthcare entity such as whether it is hypothetical or conditional.";
 ```
+
+## JS customizations
+
+### Task renames
```

Comment:
Not having a lot of context about these, aren't these general renames? Ignoring case differences and `list` operation verb prefaces, client names, client methods, model names, and model properties are generally the same across languages.

Reply:
The JS library is breaking in v6 and deviates from the other languages so it can follow the new swagger more closely. For instance, action names are noun phrases instead verb phrases. For example, the entity linking action in JS is named EntityLinkingAction but in other languages is named RecognizeLinkedEntityAction.